src/Generator/Generators/CLI/CLIHeaders.cs

```diff
@@ -798,7 +798,8 @@ public void GenerateFunction(Function function)
         public static bool FunctionIgnored(Function function)
         {
             return TypeIgnored(function.ReturnType.Type) ||
-                function.Parameters.Any(param => TypeIgnored(param.Type));
+                function.Parameters.Any(param => TypeIgnored(param.Type)) ||
+                function is Method { IsConstructor: true, Parameters: { Count: 0 }, Namespace: Class { IsValueType: true } };
```

Comment:
This is required, because CLI does not support parameterless ctors on value types, so it might remain unsupported over in CLI-land?